crates/cli-support/src/js/mod.rs

```diff
@@ -842,6 +845,16 @@ impl<'a> Context<'a> {
         Ok(())
     }
 
+    fn ts_for_init_fn() -> &'static str {
+        "\n\
+        /**\n\
+        * @param {string} module_or_path\n\
+        * @returns {Promise<any>}\n\
+        */\n\
+        export function init(module_or_path: string): Promise<any>;
```

Comment:
This might actually be best documented as `any` instead of `string` perhaps? It may be worth documenting that `string` means "go fetch this URL" (along with types like `URL` and `Request`), but everything else is "just pass this to `WebAssembly.instantiate`

Reply:
Better to list actual types that are being expected than use `any`, like:

`RequestInfo | URL | WebAssembly.Module` (or exclude `URL` for consistency with TypeScript's definition of `fetch`)

Reply:
I don't think TS has built-in support for WASM types

Reply:
Ugh yeah, looks like for now they live here: https://www.npmjs.com/package/@types/webassembly-js-api

I think it would be ok to add these types as a dependency?

Reply:
(this would also help with the return type of the function)

Reply:
Oh NVM,  `wasm-bindgen` already exports WASM types like in https://github.com/rustwasm/wasm-bindgen/blob/c5f18b609922bffce459b572daf70b2ddb190f7b/crates/cli-support/src/wasm2es6js.rs#L55